src/model/immutable/ContentBlock.js

```diff
@@ -26,7 +26,7 @@ const {List, Map, OrderedSet, Record, Repeat} = Immutable;
 const EMPTY_SET = OrderedSet();
 
 type ContentBlockConfig = {
-  key?: string,
+  key?: DraftBlockType,
```

Comment:
Just adding back, had removed this temporarily while shipping the other PR internally, but now since DraftBlockType is more permissive we wont run in the same problem again

Reply:
My mistake in here,  this should have been added to `type` not `key` but fixing it on a follow up PR https://github.com/facebook/draft-js/pull/1485